src/org/labkey/test/components/ui/entities/EntityInsertPanel.java

```diff
@@ -45,6 +48,11 @@ public WebDriver getDriver()
         return _driver;
     }
 
+    public ReactSelect entityTypeSelect()
+    {
+        return ReactSelect.finder(getDriver()).withIdStartingWith("targetEntityType").waitFor();
+    }
+
```

Comment:
Our existing test code relies on the test (or page) knowing the label text of the targetEntityType select's label.
I've chosen to add this method instead of replacing the one below (`geEntityTypeSelect(String label)`) in order to give us options about whether or not to refactor test code using it.  If it's preferable to replace older usages with this one, the refactor will be pretty easy and pleasant to do.

Reply:
I'd say deprecate the old one and we can refactor usages at our leisure while redirecting folks to use the correct method going forward.

Reply:
done